src/ctl/src/lib.rs

```diff
@@ -113,6 +113,15 @@ enum MetaCommands {
     Resume,
     /// get cluster info
     ClusterInfo,
+    /// Reschedule the parallel unit in the stream graph
+    Reschedule {
+        /// Plan of reschedule
+        #[clap(long)]
+        plan: String,
```

Comment:
Better add an example here.

Reply:
+1, which will be used for generating the command line helps automatically.